documentation/docs/tutorials/setup.md

```diff
@@ -528,3 +528,167 @@ The Grafana dashboard should be accessible at `http://<your-ip>:3000`.
 Default login credentials are:
 * `username`: admin
 * `password`: admin
+
+
+
+## Installing GoShimmer by building from source
+
+Lets first upgrade the packages on our system:
+```
+apt update && apt dist-upgrade -y
+```  
+
+### Installing RocksDB compression libraries
+
+GoShimmer uses RocksDB as its underlying db engine. That requires a few dependencies before building the project. This can be done by installing the compression libraries:
+
+```bash
+sudo apt-get install libgflags-dev libsnappy-dev zlib1g-dev libbz2-dev liblz4-dev libzstd-dev
+```  
+
+
+### Install GCC and G++
+
+This is required in order for the compilation to work properly
+
+```bash
+sudo apt install gcc g++
+```
+
+### Cloning the repository
+
+Now we have to get the GoShimmer repository by cloning it into the `/opt` directory.
+
+```bash
+cd /opt
+```
+```bash
+git clone https://github.com/iotaledger/goshimmer.git
+```
+
+### Installing Golang-go 1.16
+
+In order for the build script to work later on, we have to install the programming language Go
+
+```bash
+apt install golang-go
+````
+
+Check the go version
+
+```bash
+go version
+```
+
+If apt did not install go version 1.16+, use the method below  
+
+
+```bash
+cd /home
+```
+```bash
+wget https://dl.google.com/go/go1.16.4.linux-amd64.tar.gz
+```
+```bash
+sudo tar -xvf go1.16.4.linux-amd64.tar.gz
+```
+```bash
+sudo mv go /usr/local
+```
+```bash
+export GOROOT=/usr/local/go
+```
+```bash
+cd /opt/goshimmer
+```
+```bash
+mkdir gopath
+```
+```bash
+export GOPATH=/opt/goshimmer/gopath
+```
+```bash
+export PATH=$GOPATH/bin:$GOROOT/bin:$PATH
+```
+
+Use `go version` to check if it successfully installed golang-go 1.16
+Note that it only installs for this session. If you enter a new session then you would have to enter these commands again:
+
+```bash
+export GOROOT=/usr/local/go
+```
+```bash
+export GOPATH=/opt/goshimmer/gopath
+```
+```bash
+export PATH=$GOPATH/bin:$GOROOT/bin:$PATH
+```
+
+
+### Downloading the snapshot
+We now have to get the latest snapshot (make sure to be in the goshimmer directory)
+
+
+```bash
+sudo wget -O snapshot.bin https://dbfiles-goshimmer.s3.eu-central-1.amazonaws.com/snapshots/nectar/snapshot-latest.bin
+```
+
+### Making the node dashboard accessible
+
+Access the `config.default.json` file
```

Comment:
```suggestion
You will need to modify your goshimmer configuration file to make the Node Dashboard accessible. 
1. Make a copy of the [config.default.json](https://github.com/iotaledger/goshimmer/blob/develop/config.default.json) file and rename it config.json.
2. Using your text editor of choice, edit line 23 to change the `bindAddress` to `0.0.0.0:8081`.
3. Save your changes.
```

Reply:
I think we can skip the nano instructions as user will either know how to use it, or use any other text editor.